From 45cf1063be1f89c46e84ec3c301bdf0fdc4c8f50 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Thu, 04 Mar 2021 18:46:21 +0200
Subject: [PATCH] [SW-31170] greco: replace specs folder name to goya2

The external specs folder name is kept as goya2. In order to
overcome this issue, this patch adds change to rename specs
folder name to goya2 in case of greco HL_TARGET. This patch
will be reverted once external specs folder name is renamed to greco.

Change-Id: I2968ec19a87e031e7164aed6c31870ce26a61bd0
---

diff --git a/Makefile b/Makefile
index 8117136..83a19e9 100644
--- a/Makefile
+++ b/Makefile
@@ -22,6 +22,10 @@
 ARCBOOT_CFLAGS += -Isdk/elpsoft/src/headers
 ARCBOOT_CFLAGS += -Isdk/elpsoft/src/math
 SPECS_DEV := $(shell echo $(HL_DEV) | cut -d'_' -f1)
+ifeq ($(SPECS_DEV),greco)
+	SPECS_DEV := goya2
+endif
+
 ARCBOOT_CFLAGS += -I$(SPECS_DIR)/$(SPECS_DEV)/asic_reg
 ARCBOOT_CFLAGS += -I$(SPECS_EMBEDDED_DIR)
 ARCBOOT_CFLAGS += -DBTLVER=$(ARCBOOT_VER)
